pages/linux/loadkeys.md

```diff
@@ -0,0 +1,32 @@
+# loadkeys
+
+> Load the kernel keymap for the console.
+> More information: <https://manned.org/loadkeys>.
+
+- Load a default keymap:
+
+`loadkeys --default`
+
+- Create a kernel source table:
+
+`loadkeys --mktable`
+
+- Create a binary keymap:
+
+`loadkeys --bkeymap`
+
+- Search and parse keymap without action:
+
+`loadkeys --parse`
+
+- Suppress all normal output:
```

Comment:
```suggestion
- Load the keymap suppressing all output:
```

Reply:
Thanks for the clarification and helping me out. I have updated the description.